pandas/io/parsers.py

```diff
@@ -709,7 +707,6 @@ def read_fwf(
     infer_nrows=100,
     **kwds,
 ):
-
```

Comment:
same

Reply:
I was unsure if this was supposed to be 2 lines or just one.  I went with two since you said top level functions require 2 lines.  I am coming from Java so python styling is still strange to me!

Reply:
I was unsure if this was supposed to be 2 lines or just one.  I went with two since you said top level functions require 2 lines.  I am coming from Java so python styling is still strange to me!

Reply:
This is the Python style convention used by (mostly) everybody: https://www.python.org/dev/peps/pep-0008/

Black should take care of it I guess. The main thing is to check `git diff upstream/master` (or the `Files Changed` tab in GitHub, and make sure you don't change things unrelated to what you're addressing.